contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -47,6 +47,11 @@ ConfigManager::ConfigManager(
     }
   }
 
+  if (global_context_->server_config()->has_init_service_configs()) {
```

Comment:
ApiManager creates the instance of ConfigManager only after the deployment was successful.
It would be safe to copy the value here.